src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -16,17 +17,24 @@ const chatMessageStyles: ComponentSlotStylesInput<ChatMessageProps, ChatMessageV
     maxWidth: v.messageWidth,
     wordBreak: 'break-word',
     wordWrap: 'break-word',
+    ':focus': {
+      outline: 'none',
+      '& .ui-chat__message__messageBody': {
+        outline: `.2rem solid ${siteVariables.brand}`,
+      },
```

Comment:
to not write classes manually, can we do smth like?

```js
':focus': {
   outline: 'none',

   messageBody: {
      outline: `.2rem solid ${siteVariables.brand}`,
   }
}
```

Reply:
why would this work, there is no style / DOM element named `messageBody`. am I missing something, @sophieH29 ?

Reply:
I meant to use already defined class variable that you have here ```className={cx('ui-chat__message__messageBody', classes.messageBody)}```
Wasn't sure if it is possible, that's why I asked :)

Reply:
we cannot, CSS in JS styles cannot be  aware of CSS classes **they** generate;
`messageBody` class is generated from the styles in `messageBody` lot in `src/themes/teams/components/Chat/chatMessageStyles.ts` so it cannot be referenced yet; 

moreover, we don't have access to class names anyway, you we're referring to `messageBody ` as the value of the class name itself in `src/themes/teams/components/Chat/chatMessageStyles.ts` but it's just the name of the variable storing the class name (the value is something generated by Fella).

hope it makes sense